### ISSUE TITLE
aria-selected in multi-selectable treeviews, set to false or remove attribute

### ISSUE THREAD

**Issue by jongund** [w3c/aria-practices#745]
When a item in a tree view is not selected, should it set aria-selected="false" or remove the aria-selected attribute.   It seems to make a difference in screen readers.   The new multi-selectable tree views has two examples, one sets aria-selected="false", the other removes the aria-selected attribute when an itemin the tree is not selected.

**Comment by jnurthen** [comment]
If something is multi-selectable (aria-multiselectable=true) then aria-selected=false means that it can be selected but it is not currently selected. 
If aria-selected is not present it means that the element cannot be selected.

**Comment by mcking65** [comment]
@jnurthen is correct. This is documented in the pattern as follows:

>* if the tree supports multiple selection: 
>    * All selected nodes have aria-selected set to true.
>    * All nodes that are not selected have aria-selected set to false.

A nit, it could be possible that some nodes are not selectable. So, that language is not as precise as it could be. I am making a minor revision with commit 05faf11. I imagine there are other patterns that support multi-selection where a similar oversight is present. I'm not hunting for them at this hour though. If someone would like to hunt them down and raise an issue, that'd be nice.

**Comment by mcking65** [comment]
Question answered, build was successful, closing. @jongund, feel free to re-open if not sufficiently clear.
